IPython/core/oinspect.py

```diff
@@ -587,7 +587,21 @@ def format_mime(self, bundle):
         return bundle
 
     def _get_info(self, obj, oname='', formatter=None, info=None, detail_level=0):
-        """Retrieve an info dict and format it."""
+        """Retrieve an info dict and format it.
+        
+        Parameters
+        ==========
+
+        obj: any
+            Object to inspect and return info from
+        oname: str
+            Passed to self._info as is
```

Comment:
As `self._info` as a private method, I think this should explain the parameter rather than referring to another method. Though I'm still not entirely sure why we're deprecating `self.info()`.